elkscmd/levee/Makefile

```diff
@@ -12,7 +12,7 @@ include $(BASEDIR)/Make.rules
 
 ###############################################################################
 
-CFLAGS = -ansi -O -0 -DSIZE=16000L
+LOCALFLAGS=-DSIZE=16000L
 
 xCC=gcc
```

Comment:
Shell variables must be in all caps.

Reply:
These `x*` are legacy and unused ones. Cleanup done for the next commit.